examples/vault-variable-test/spin.toml

```diff
@@ -0,0 +1,19 @@
+spin_manifest_version = 2
```

Comment:
I might be missing something but the manifest doesn't seem to define the variables?